tests/e2e/e2e_test.go

```diff
@@ -49,6 +50,25 @@ func (s *IntegrationTestSuite) TestConcentratedLiquidity() {
 	s.Require().Equal(concentratedPool.GetTickSpacing(), tickSpacing)
 	s.Require().Equal(concentratedPool.GetPrecisionFactorAtPriceOne(), sdk.NewInt(precisionFactorAtPriceOne))
 	s.Require().Equal(concentratedPool.GetSwapFee(sdk.Context{}), sdk.MustNewDecFromStr(swapFee))
+
+	// Initialize 2 more nodes
+	node2, err := chainA.GetNodeAtIndex(1)
+	s.Require().NoError(err)
+	node3, err := chainA.GetNodeAtIndex(2)
+	s.Require().NoError(err)
```

Comment:
It doesn't matter what node you're submitting it from. I suggest using only one node for simplicity